components/Table.tsx

```diff
@@ -0,0 +1,61 @@
+import React from "react"
+import { 
+  useTable,
+  Column,
+} from "react-table"
+import { 
+  Table as ChakraTable,
+  Thead,
+  Tbody,
+  Tr,
+  Th,
+  Td,
+} from "@chakra-ui/react"
+
+export interface TableProps {
+  columns: Column<object>[]
+  data: object[]
+}
+
+export const Table = ({columns, data}: TableProps ): JSX.Element =>  {
+  const {
+    getTableProps,
+    getTableBodyProps,
+    headerGroups,
+    rows,
+    prepareRow,
+  } = useTable({
+    columns,
+    data,
+  })
+
+  return (
+    <ChakraTable {...getTableProps()}>
```

Comment:
According to the [React Table's docs](https://react-table.tanstack.com/docs/api/useTable#instance-properties), we need to use different functions returned by the `useTable()` hook to resolve props for different components within the table (the overall table wrapper, rows, headers, etc.).